svgdigitizer/electrochemistry/cv.py

```diff
@@ -248,42 +248,23 @@ def get_axis_unit(cls, unit):
             >>> CV.get_axis_unit(unit)
             Unit("uA / cm2")
 
-        """
-        unit_typos = {
-            "uA / cm2": [
-                "uA / cm2",
-                "uA / cm²",
-                "µA / cm²",
-                "uA/cm2",
-                "uA/cm²",
-                "µA/cm²",
-                "µA cm⁻²",
-                "uA cm-2",
-            ],
-            "mA / cm2": [
-                "mA / cm2",
-                "mA / cm²",
-                "mA cm⁻²",
-                "mA/cm2",
-                "mA/cm²",
-                "mA cm-2",
-            ],
-            "A / cm2": ["A / cm2", "A/cm2", "A cm⁻²", "A cm-2"],
-            "uA": ["uA", "µA", "microampere"],
-            "mA": ["mA", "milliampere"],
-            "A": ["A", "ampere", "amps", "amp"],
-            "mV": ["milliV", "millivolt", "milivolt", "miliv", "mV"],
-            "V": ["V", "v", "Volt", "volt"],
-            "V / s": ["V s-1", "V/s", "V / s"],
-            "mV / s": ["mV / s", "mV s-1", "mV/s"],
-        }
+            >>> unit = 'uA cm⁻²'
+            >>> CV.get_axis_unit(unit)
+            Unit("uA / cm2")
 
-        for correct_unit, typos in unit_typos.items():
-            for typo in typos:
-                if unit == typo:
-                    return u.Unit(correct_unit)
 
-        raise ValueError(f"Unknown Unit {unit}")
+        >>> unit = 'milliV'
+        >>> CV.get_axis_unit(unit) # doctest: +IGNORE_EXCEPTION_DETAIL
+        Traceback (most recent call last):
+        ...
+        ValueError: Astropy was not able to convert the given string (milliV) into a meaningful astropy unit. Please review the unit string.
+        """
+        try:
+            return u.Unit(unit)
+        except ValueError as err:
+            raise ValueError(
+                f"Astropy was not able to convert the given string ({unit}) into a meaningful astropy unit. Please review the unit string."
+            ) from err
```

Comment:
Actually, the astropy exception is kind of nice so we might one to keep it?

```suggestion
        return u.Unit(u)
```

It says "'milliV' did not parse as unit: At col 0, milliV is not a valid unit. Did you mean milliVolt or millivolt? If this is meant to be a custom unit, define it with 'u.def_unit'. To have it recognized inside a file reader or other code, enable it with 'u.add_enabled_units'. For details, see https://docs.astropy.org/en/latest/units/combining_and_defining.html".

Sure, the col 0 is confusing and the final part is also weird. But the "did you mean" actually works here.

Reply:
So, should we completley eliminate get_axis_unit()?